contrib/hbwin/hbwin.hbp

```diff
@@ -29,7 +29,7 @@ mapi.c
 oemansi.c
 olecore.c
 oleuuid.c{watcom}
-hbolesrv.c
+{HBWIN_HAS_OLESRV}hbolesrv.c
```

Comment:
> 2) Add xhb.hbc. xhb has a dependency on hbwin, which in turn defines the symbol DllMain.

If you need to link `xhb.hbc` (or any other `.hbc`) to a `.dll` and wish that its dependencies be 
linked dynamically (forming a chain of `.dll`'s) there is already a facility for that: set the envvar 
`_HB_DYNSUFFIX` to the value returned by `hb_libSuffix()` (f.e. `-32` for 32-bit mingw) and 
`hbwin.hbc` and others will be linked as `.dll`s as opposed to statically linked copies.